index.js

```diff
@@ -1,29 +1,51 @@
 module.exports = {
-  extends: ['eslint-config-airbnb'].map(require.resolve),
+  env: {
+    'jest/globals': true,
+    browser: true,
+  },
+  extends: [
+    'eslint-config-airbnb',
+    'eslint-config-prettier',
+    'eslint-config-prettier/react',
+  ].map(require.resolve),
   parser: 'babel-eslint',
-  plugins: [
-    "mocha"
-  ],
+  plugins: ['jest', 'prettier'],
   rules: {
     'arrow-body-style': 0,
     'arrow-parens': ['error', 'as-needed'],
-    'camelcase': ['error'],
+    camelcase: ['error'],
+    'no-prototype-builtins': 0,
+    'import/prefer-default-export': 0,
+    'import/no-named-default': 0,
+    'import/first': 0,
     'import/no-unresolved': 0,
-    'max-len': 0,
-    'no-param-reassign': 0,
-    'no-underscore-dangle': [2, { 'allowAfterThis': true }],
-    'no-unused-vars': ['error', { vars: 'all', args: 'after-used' }],
+    'import/no-extraneous-dependencies': 0,
+    'no-underscore-dangle': [2, { allowAfterThis: true }],
+    semi: [2, 'never'],
+    'no-plusplus': ['warn'],
```

Comment:
gross, whY?

<img width="463" alt="screen shot 2017-10-17 at 3 30 09 pm" src="https://user-images.githubusercontent.com/4994330/31692865-7526f694-b350-11e7-870d-c8d5a018fb6f.png">

Reply:
change to map

Reply:
fun fun functions

Reply:
the rule has nothing to do with the `for` statement but disallows the use of the `++` operator.  not seeing a point in forbidding this syntax.

Reply:
oh im just saying that this specific case id rather refactor this to use a map

Reply:
TBH im fine with a lint rule disallowing for loops

Reply:
> oh im just saying that this specific case id rather refactor this to use a map
  
definitely 

> TBH im fine with a lint rule disallowing for loops

😮  ... actually, i'd LGTM it :)

Reply:
> TBH im fine with a lint rule disallowing for loops

Let's do it

Reply:
:D